src/test/java/org/folio/logintest/CredentialExistenceTest.java

```diff
@@ -87,15 +87,8 @@ public static void setup(final TestContext context) {
 
   @AfterClass
   public static void teardown(TestContext context) {
-    PostgresClient.getInstance(vertx, TENANT).delete(TABLE_NAME_CREDENTIALS, new Criterion(), event -> {
-      if (event.failed()) {
-        context.fail(event.cause());
-      }
-    });
-
-    vertx.close(context.asyncAssertSuccess(res -> {
-      PostgresClient.stopEmbeddedPostgres();
-    }));
+    PostgresClient.stopEmbeddedPostgres();
```

Comment:
Stopping embedded postgres is correct.
Starting embedded postgres is no longer needed since RMB 26: https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-26
Code that starts embedded postgres should be removed.
If a developer doesn't use embedded postgres deleting TABLE_NAME_CREDENTIALS is useful.

Reply:
Thanks. Removed the redundant starting db call.